tests/auxtel/test_build_pointing_model.py

```diff
@@ -0,0 +1,274 @@
+# This file is part of ts_externalscripts
+#
+# Developed for the LSST Telescope and Site Systems.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+
+import types
+import unittest
+import contextlib
+import unittest.mock
+
+import numpy as np
+
+from lsst.ts.standardscripts import BaseScriptTestCase
+from lsst.ts.externalscripts import get_scripts_dir
+from lsst.ts.externalscripts.auxtel.build_pointing_model import BuildPointingModel
+from lsst.ts.observatory.control.utils.enums import RotType
+
+
+class TestBuildPointingModel(BaseScriptTestCase, unittest.IsolatedAsyncioTestCase):
+    def setUp(self) -> None:
```

Comment:
What does the function annotation of "None" do?

Reply:
That is python type annotation.. just specifying that the `setUp` method does not return anything (or return `None`, which is the same thing).